src/main/java/org/osiam/resources/provisioning/update/ExtensionUpdater.java

```diff
@@ -113,7 +113,8 @@ private void updateExtensionField(Entry<String, Extension> extensionEntry, UserE
             try {
                 extensionEntitiyField = extensionEntity.getFieldForName(fieldName, true);
             } catch (NoSuchElementException e) {
-                throw new NoSuchElementException("Could not update the extension \"" + urn + "\".", e);
+                LOGGER.info("Update Extension failed: ", e.getMessage());
```

Comment:
I'm happy to increase the level to warn. But we don't really have a logging strategy yet. Maybe it's time to talk about one?